src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import "./SearchForm.css";
+
+const SearchForm = ({ handleSearch }) => { 
+  const [query, setQuery] = useState("");
+  const [error, setError] = useState("");
+
+  // Handle form submission
+  const onSubmit = (e) => {
+    e.preventDefault();
+    if (!query.trim()) {
+      setError("Please enter a keyword");
+    } else {
+      setError("");
+      handleSearch(query);  // Pass the search query to App.jsx
+      setQuery("");  //Clear the search field after submission
+    }
+  };
+
+  // Clear error message when user types in the search input
+  const handleInputChange = (e) => {
+    setQuery(e.target.value);
+    if (error) {
+      setError("");  // Clear error when the user starts typing again
+    }
+  };
+
+  return (
+    <form onSubmit={onSubmit} className="search-container">
+      <input
+        type="text"
+        value={query}
```

Comment:
Good job that you don't forget about `value` in the inputs if you have `onChange`